filebeat/module/system/auth/_meta/fields.yml

```diff
@@ -24,6 +24,54 @@
       description: >
         The Unix user that this event refers to.
 
+    - name: pam
+      type: group
+      description: >
+        Fields specific to PAM messages.
+      fields:
+      - name: module
+        description: >
+          The name of the PAM module.
+      - name: service
+        description: >
+      - name: type
+        description: >
+          The PAM type.
+      - name: result
+        description: >
+          The PAM result text, e.g. "authentication success".
+      - name: logname
+        description: >
+          The PAM logname.
+      - name: uid
+        type: long
+        description: >
+          The UID of the PAM process.
+      - name: euid
+        type: long
+        description: >
+          The EUID of the PAM process.
+      - name: tty
+        description: >
+          The TTY of the PAM process.
+      - name: ruser
+        description: >
+          The remote username.
+      - name: rhost
+        type: ip
+        description: >
+          The remote host IP address.
+      - name: user
+        description: >
+          The username.
```

Comment:
What is the difference between this user and ruser? is this the user of the PAM process?

Reply:
It appears that logname is the user of the PAM calling process.  ruser is the user making the request.  user is the user that the process is trying to authenticate to.  I've updated the docs.